tomcat/base_test.go

```diff
@@ -288,10 +288,12 @@ func testBase(t *testing.T, context spec.G, it spec.S) {
 	context("$BP_TOMCAT_ENV_PROPERTY_SOURCE_DISABLED", func() {
 		it.Before(func() {
 			Expect(os.Setenv("BP_TOMCAT_ENV_PROPERTY_SOURCE_DISABLED", "true")).To(Succeed())
+			Expect(os.Setenv("BPI_TOMCAT_ADDITIONAL_JARS", "/layers/test-buildpack/foo/bar.jar")).To(Succeed())
```

Comment:
I think this change deserves its own unit test, not just an additional assertion to an existing test

Reply:
Fair point. That was even my initial thought. But those tests had so much setup that I added it. I will adapt it.

Reply:
The diff looks a bit weird, but I moved the creation of the resources to the `it.Before` since it was always the same and is not really relevant for those test cases.

Reply:
fair enough! thanks for the update!